lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -87,6 +90,39 @@ class DataDownload
           @$reports_request_response.text data['status']
           $(".msg-confirm").css({"display":"block"})
 
+    @$list_issued_certificate_table_btn.click (e) =>
+      url = @$list_issued_certificate_table_btn.data 'endpoint'
```

Comment:
@mattdrayer  @andy-armstrong ^ ^    - are these changes in this file are acceptable ?